### PR TITLE
DFR-1999 shared redis for front end

### PR DESCRIPTION
**Pull request by gmikerlemos**
https://tools.hmcts.net/jira/browse/DFR-1999

### CODE REVIEW

**Review by timja — COMMENTED** [review]
I think you want this as it's already deployed in div-petitioner-frontend you will need to deploy a new one and swapover. you would also need to store the connection string as well i think

File: main.tf [hunk 1]
```diff
@@ -8,3 +8,15 @@ resource "azurerm_resource_group" "rg" {
 
   tags = var.common_tags
 }
+
+module "redis-cache" {
```
Comment (timja) [inline main.tf:12]: don't you need to store the connection string in a key vault as well?

Comment (gmikerlemos) [reply main.tf:12]: yes, added

**Review by timja — APPROVED** [review]

**Review by viktoriyabogdanova — COMMENTED** [review]
You are missing the parameter name 
name     = "${var.product}-${var.component}-${var.env}"

**Comment by gmikerlemos** [comment]
> You are missing the parameter name
> name = "${var.product}-${var.component}-${var.env}"

WHy do i need that?

**Pull request merged**